README.md

```diff
@@ -141,12 +141,22 @@ sudo apt-get install \
 
 In the folly directory, run:
 ```
-  mkdir _build && cd _build
+  mkdir _build
   cmake ..
   make -j $(nproc)
   make install
 ```
 
+If you receive an error like:
+
+> CMake Error at cmake_install.cmake:41 (file):
+> file INSTALL cannot copy file
+> "/home/[yourname]/[folly-version]/folly/libfolly.a" to
+> "/usr/local/lib/libfolly.a".
+
+then you [need to use `sudo make install`](https://askubuntu.com/a/954877)
+as the last step instead of just `make install`.
```

Comment:
Not sold on this whole comment needing to exist. Installing things into the system is well-known to require admin privileges or the root user, and sudo is the well-known way to get those on Ubuntu.

Of course there are many people who don't know this, but it's not really about folly and it's more about getting familiar with the operating system.

At best, we can have an in-line comment suggesting either to use sudo or to use DESTDIR but without offering any explanation. E.g.:

```
  make install # with either sudo or DESTDIR as necessary
```

Reply:
Alright, I will make the change.